readme.md

```diff
@@ -154,6 +154,22 @@ alfy.cache.get('unicorn');
 //=> '🦄'
 ```
 
+##### maxAge
+
+The `set` method of this instance accepts an optional third argument where you can provide a `maxAge` option. `maxAge` is
+the number of milliseconds the value is valid in the cache.
+
+Example:
+
+```js
+alfy.cache.set('foo', 'bar', {maxAge: 5000});
+
```

Comment:
Do a `alfy.cache.get('foo');` and show the value before the delay too. For clarity.